content/en/docs/home/_index.md

```diff
@@ -41,20 +41,20 @@ cards:
   description: "Look up common tasks and how to perform them using a short sequence of steps."
   button: "View Tasks"
   button_path: "/docs/tasks"
-- name: training
-  title: "Training"
-  description: "Get certified in Kubernetes and make your cloud native projects successful!"
-  button: "View training"
-  button_path: "/training"
 - name: reference
   title: Look up reference information
   description: Browse terminology, command line syntax, API resource types, and setup tool documentation.
   button: View Reference
   button_path: /docs/reference
+- name: training
+  title: "Training"
+  description: "Get certified in Kubernetes and make your cloud native projects successful!"
+  button: "View training"
+  button_path: "/training"
 - name: contribute
-  title: Contribute to the docs
+  title: Contribute to Kubernetes
   description: Anyone can contribute, whether you're new to the project or you've been around a long time.
-  button: Contribute to the docs
+  button: Find out how to help
```

Comment:
nit: Contribute to the docs is clear where 'find out how to help' seems to indicate that you are searching for how to do something unspecified

Reply:
Here's how that renders:
![sample rendering](https://github.com/kubernetes/website/assets/22591623/19f09c73-1dc5-45f2-9528-e5ea17d3ef05)

Is the button clear enough in context? I could change it to “Find out how to contribute” but these are almost synonymous. I specifically want to avoid restricting things to people who want to help with docs; people who want to help in other ways are also welcome to click.

Reply:
@sftim But the button path is still https://kubernetes.io/docs/contribute/, so it might cause confusion for folks looking to contribute beyond docs but are being directed to our docs contribute page. I agree with @kbhawkey that the former is more clear

Reply:
Top of our docs contribute page:
> Kubernetes welcomes improvements from all contributors, new and experienced!
> 
> **Note:**
> 
> To learn more about contributing to Kubernetes in general, see the [contributor documentation](https://www.kubernetes.dev/docs/).
> 
> You can also read the [CNCF](https://cncf.io/) [page](https://contribute.cncf.io/contributors/projects/#kubernetes) about contributing to Kubernetes.

That's why I'm happy to link there. I'd be willing to make a custom shortcode to make the point more pointedly, even.

Reply:
BTW, why not link directly to https://www.kubernetes.dev/docs/? Because https://www.kubernetes.dev/ is only localized to English, and the main website has lots of localizations.

Reply:
(does that make sense / should I nevertheless make changes?)